### DISCUSSION TITLE
怎样合并github上的两个库

### DISCUSSION THREAD

**Discussion by PaperL** [ACM-Class-2020/DataStructure#9] [Q&A] [answered]
有两个库，希望合并成一个，并且能保留两个库的commit记录

**Comment by Marcythm** [comment, marked as answer]
https://medium.com/altcampus/how-to-merge-two-or-multiple-git-repositories-into-one-9f8a5209913f

https://techtldr.com/git-merg-two-or-more-repositories-and-keeping-history/
